app/models/condition.rb

```diff
@@ -189,8 +192,9 @@ def self._subst_find(rec, inputs, expr)
 
     if checkmode == "count"
       e = check.gsub(/<count>/i, list.length.to_s)
+      left, operator, right = e.split
       MiqPolicy.logger.debug("MIQ(condition-_subst_find): Check Expression after substitution: [#{e}]")
-      result = !!proc { $SAFE = 3; eval(e) }.call
+      result = !!left.to_f.send(operator, right.to_f)
```

Comment:
The string is built by the UI and it looks something like this  <count> = 5. The string <count> is a place holder for the count of results found in the "find" portion of the expression. We're substituting <count> with the actual count of find results on line 194. The value on the right side is input by the end user. The UI only allows for input of a numeric value. The operators are limited to =, !=, <, > <=, >= by the UI.